changelogs/fragments/2136-zos_unarchive_skip_encoding_support.yml

```diff
@@ -0,0 +1,3 @@
+trivial:
+   - zos_unarchive - Adds the skip_encoding parameter to the unarchive module.
```

Comment:
Lets keep it similar to the archive one: "Adds support for skipping encoding in zos_unarchive module.
    This allows users to skip encoding for certain files after unarchiving them."